src/coreclr/src/tools/crossgen2/crossgen2/Properties/Resources.resx

```diff
@@ -258,6 +258,9 @@
   <data name="WarningOverridingOptimizeSpace" xml:space="preserve">
     <value>Warning: overriding -Ot with -Os</value>
   </data>
+  <data name="CustomPESectionAlignmentOption" xml:space="preserve">
+    <value>Use custom alignment for PE sections in PE generated PE file</value>
```

Comment:
```suggestion
    <value>PE custom PE for PE PE in PE generated PE file</value>
```

Reply:
Lol, I think I have an extra PE. I've removed one of them. Sadly, I think this switch is destined to be used only by a very small set of customers.

Reply:
Tell more people about it ... we should use this feature!